cloudbuild/python-scripts/insert_dynamic_partitions.py

```diff
@@ -0,0 +1,146 @@
+"""
+Python script to dynamically partitions to a BigQuery partitioned table.
+"""
+import argparse
+import random
+from collections.abc import Sequence
+import datetime
+import threading
+import time
+from absl import app
+from utils import utils
+
+
+def wait():
+    print(
+        'Going to sleep, waiting for connector to read existing, Time:'
+        f' {datetime.datetime.now()}'
+    )
+    # This is the time connector takes to read the previous rows
+    time.sleep(2.5 * 60)
```

Comment:
Shouldn't the wait be determined by frequency of new partition discovery? It does not matter when the previous rows were read, because read streams are created for the table's snapshot at moment of read session creation. So, we need to insert new entries after the previous read session was created and before the next partition discovery. We should also mention that in the comment.

Reply:
Correct. This wait is to make sure that any collision does not happen. Such that new rows are generated only after the previous read session was created. We wait for the next read separately.

Reply:
Then let's change the comment accordingly, or remove it. The current description is misleading.